binder/appendix.txt

```diff
@@ -0,0 +1,5 @@
+USER root
+# Enable user to edit software environment via a writeable tmpfs overlay
+# in Singularity.
+RUN chmod a+w -R /srv
```

Comment:
Maybe `chown`-ing to `${NB_USER}` will eliminate the need for a world-writable directory? Not sure if there are any limitations though.

Reply:
The `${NB_USER}`, known at build time, is `jovyan`, but the container is run as whatever BNL `$USER` is logged in, which will have a different uid.